src/super_context.ml

```diff
@@ -901,6 +902,24 @@ module PP = struct
           mli) in
     { m with impl ; intf }
 
+  let uses_ppx_driver ~pps =
+    match Option.map ~f:Pp.to_string (List.last pps) with
+    | Some "ppx_driver.runner" -> true
+    | Some _ | None -> false
+
+  let promote_correction ~uses_ppx_driver fn build =
+    if not uses_ppx_driver then
+      build
+    else
+      Build.progn
+        [ build
+        ; Build.return
+            (A.promote_if
```

Comment:
Is it really worth introducing a module alias just for this one use?

Reply:
It's just that super_context defines an Action module as well, so we get a name clash